templates/_index.md.template

```diff
@@ -0,0 +1,15 @@
+---
+title: "Tekton Documentation Vault"
+linkTitle: "Vault"
+---
+
+This section keeps the documentation of past versions of Tekton components.
```

Comment:
minor nit: suggest "provides" instead of "keeps".

Reply:
Done :) Thanks!